tools/linter/adapters/s3_init_config.json

```
@@ -21,17 +21,13 @@
         }
     },
     "clang-tidy": {
-        "Darwin-i386": {
-            "download_url": "https://oss-clang-format.s3.us-east-2.amazonaws.com/macos-i386/15.0.6/clang-tidy",
```

```suggestion
            "download_url": "https://oss-clang-format.s3.us-east-2.amazonaws.com/macos-i386/19.1.4/clang-tidy",
```
Tested and the download url works, just also update the hash